### ISSUE TITLE
How to workaround "SC2086" for shells other than bash.

### ISSUE THREAD

**Issue by resuna** [koalaman/shellcheck#362]
All the world isn't bash, and scripts that are intended to be portable across shells can't depend on bash features. Therefore the workaround on https://github.com/koalaman/shellcheck/wiki/Sc2086 is incomplete.

Suggestion: a tagged comment allowing variables to be flagged safe for unquoted expansion.

Eg:

    #!/bin/sh
    
    FLAGS="-av --delete --delete-excluded --exclude=.DS_Store"
    
    [...]
    
    rsync $FLAGS ~/Calibre\ Library/. "$D/ebooks/Calibre/."
               ^––SC2086 Double quote to prevent globbing and word splitting.

Workaround:

    #!/bin/sh
    
    #pragma safequote FLAGS
    FLAGS="-av --delete --delete-excluded --exclude=.DS_Store"
    
    [...]
    
    rsync $FLAGS ~/Calibre\ Library/. "$D/ebooks/Calibre/."

**Comment by myint** [comment]
I'm not sure where the documentation for this is, but I think what you want is:

```bash
# shellcheck disable=SC2086
rsync $FLAGS ~/Calibre\ Library/. "$D/ebooks/Calibre/."
```

**Comment by resuna** [comment]
I don't want to disable it, I just want to disable known false positives.

**Comment by koalaman** [comment]
Disabling only affects one command immediately following it, do it's safe to use for things like this

I've updated the wiki to say that arrays are bash/ksh specific, and suggest using a function for POSIX
